packages/manager/src/features/Volumes/VolumeCreate.tsx

```diff
@@ -240,6 +243,15 @@ export const VolumeCreate = () => {
 
   const { config_id, linode_id } = values;
 
+  const { data: linode } = useLinodeQuery(
+    linode_id ?? -1,
+    isBlockStorageEncryptionFeatureEnabled
```

Comment:
Late to this PR, but noticed a minor issue that can be resolved in a later change:

To prevent an API call for `linodes/instances/-1`
```suggestion
    isBlockStorageEncryptionFeatureEnabled && linode_id !== null
```